run_flake8.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
-EXCLUDE=`cat flake8_exclude.txt`
+EXCLUDE=$(cat flake8_exclude.txt)
```

Comment:
what is the difference now?

Reply:
none really. was just remarked by the code review, and I did not want to disagree.